packages/features/ee/workflows/lib/reminders/templates/emailReminderTemplate.ts

```diff
@@ -3,6 +3,8 @@ import { APP_NAME } from "@calcom/lib/constants";
 import { TimeFormat } from "@calcom/lib/timeFormat";
 import { WorkflowActions } from "@calcom/prisma/enums";
 
+require("dayjs/locale/en");
+
```

Comment:
I don't think it's a right way to fix that

Reply:
Okay, i will find some way to get it more structured or if u can suggest something on this that will be really helpful.

Reply:
Hi @anikdhabal, Corrected it, Can u please check?